xdsl/dialects/llvm.py

```diff
@@ -15,6 +15,8 @@
     from xdsl.parser import BaseParser
     from xdsl.printer import Printer
 
+min32 = -2147483648
```

Comment:
I think moving this into the `GEPOp` would also make it more clear where it belongs to.

Reply:
When moving it into GEPOp, we get error:

```
xdsl.utils.exceptions.PyRDLOpDefinitionError: SSADummy is neither a function, or an operand, result, region, or attribute definition. Operands should be defined with type hints of Annotated[Operand, <Constraint>], results with Annotated[OpResult, <Constraint>], regions with Region, and attributes with OpAttr[<Constraint>]
```

Reply:
Ah, that's because `pyrdl` disallows non-irdl fields on classes... I forgot about that. 

I guess I'd name the variable `GEP_USE_SSA_VAL` or something with a docstring specifying what it does, like:

```python
GEP_USE_SSA_VAL = -2147483648
"""
This is used in the getelementptr index list to signify that an ssa value 
should be used for this index.
"""
```

Reply:
Sounds good